### ISSUE TITLE
Handling of bias inconsistent

### ISSUE THREAD

**Issue by marcoancona** [VigneshSrinivasan10/interprettensor#2]
Hello, thanks for sharing this code. I noticed some inconsistencies between Dense (Linear) and Convolutional layers.
In the former you add the bias when computing zs (linear.py, line 101):

```python
Zs = tf.expand_dims(tf.reduce_sum(Z, 1), 1) + tf.expand_dims(tf.expand_dims(self.biases, 0), 0)
```

but for Convolutional layer the adding of bias is commented out (convolution.py, line 192):

```python
Zs = tf.reduce_sum(Z, [3,4,5], keep_dims=True)  #+ tf.expand_dims(self.biases, 0)
```

This looks like an inconsistency to me, unless I am missing something in understanding LRP itself.

**Comment by VigneshSrinivasan10** [comment]
Hi @marcoancona, Apologies for the delay. The inconsistency has been fixed now. 
We also have additional features like batch normalization now. 

Good luck with your experiments. 
Best,
